lib/catalog/martyrology.ts

```diff
@@ -978,9 +978,6 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Declan',
       titles: [Titles.Bishop],
     },
-    dedication_of_consecrated_churches: {
-      name: 'Dedication of Consecrated Churches whose date of Consecration is unknown',
-    },
```

Comment:
Why is this removed? If we should list event celebrations in `martyrology.ts`, we should keep `dedication_of_consecrated_churches`.

```suggestion
    dedication_of_consecrated_churches: {
      name: 'Dedication of Consecrated Churches Whose Date of Consecration is Unknown',
    },
```

Reply:
This one has been removed because there are neither celebrations related to `dedication_of_consecrated_churches` nor any martyrology reference to a `dedication_of_consecrated_churches` entity.

Furthermore, the martyrology is not intended for such types of celebrations. It should only contain elements related to individuals (saints or blessed individuals).

Reply:
Consider this a request not blocking this PR from merging: I still want see event celebrations meatadata included in romcal, however, I don’t if they are moved into a separate file (e.g. `events.ts`).

Reply:
You are mixing the entities from the martyrology and those from the calendar definitions. Currently, there are two calendar entities (`dedication_of_consecrated_churches_on_october_25` and `dedication_of_consecrated_churches_on_last_sunday_of_october`) that are well included in the data returned by Romcal. 

Just FYI, `dedication_of_consecrated_churches` corresponds only to the translation key defined from the `customLocaleId` property of the two definitions. So, here in this case (in the martyrology file), this entity is indeed unnecessary.

By the way, we already have an open issue https://github.com/romcal/romcal/issues/136, where I haven't had the chance to respond yet.